.github/workflows/dependabot-pr.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot PR
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependabot:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    if: github.triggering_actor == 'dependabot[bot]'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.PAT }}
+
+      - name: update code
+        run: |
+          input="${{ github.event.pull_request.title }}"
+
+          regex="[Bb]ump (.*) from (.*) to (.*)"
+          if [[ ! "$input" =~ $regex ]]; then
+            echo "unexpected pattern"
+            exit 1
+          fi
+
+          crate_name="${BASH_REMATCH[1]}"
+          old_version="${BASH_REMATCH[2]}"
+          new_version="${BASH_REMATCH[3]}"
+          echo "crate_name: $crate_name, old_version: $old_version, new_version: $new_version"
+
+          ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name:$old_version --precise $new_version
+          ./scripts/cargo-for-all-lock-files.sh -- tree
+
+      - name: push
+        run: |
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
```

Comment:
custom name and email aren't allowed? say, "dependabot-github-action". while `41898282` is still `git grep`-able from git log to reach to this script, i think more descriptive name helps.

Reply:
oh okay! maybe I switch them back to the original ones! 
https://github.com/anza-xyz/agave/blob/9035690e40c0dbffc83ef8c2aa168078853359d7/ci/dependabot-pr.sh#L23
https://github.com/anza-xyz/agave/blob/9035690e40c0dbffc83ef8c2aa168078853359d7/ci/dependabot-pr.sh#L29

Reply:
what do you think if we just use the dependabot itself :trollface: [14a411d](https://github.com/anza-xyz/agave/pull/1466/commits/14a411d40c05e3be3a352ac76f51fffc622fa1b9)